types/backbone/index.d.ts

```diff
@@ -554,7 +554,7 @@ declare namespace Backbone {
          * For assigning events as object hash, do it like this: this.events = <any>{ "event:selector": callback, ... };
          * That works only if you set it in the constructor or the initialize method.
          */
-        events(): EventsHash;
+        events: _Result<EventsHash>;
```

Comment:
I somewhat agree with this change, but it really needs to have a test going with it.

Reply:
Sure, I'll add a test. 

Is the reason you 'somewhat' agree because of the method declaration issue?

Reply:
Added tests.

Reply:
I agree in principle but this change breaks the other half of use cases. That's why I'm not super-duper enthousiasmic about it.

Reply:
Yeah, as far as I know, there isn't a way to allow both, unfortunately.